extensions/eclipse-che-theia-git-provisioner/package.json

```diff
@@ -20,7 +20,9 @@
     "@theia/preferences": "next",
     "ini": "^1.3.5",
     "nsfw": "^2.1.2",
-    "@eclipse-che/theia-user-preferences-synchronizer": "0.0.1"
+    "@eclipse-che/theia-user-preferences-synchronizer": "0.0.1",
+    "@kubernetes/client-node": "^0.12.1",
+    "@eclipse-che/theia-remote-impl-che-server": "0.0.1"
```

Comment:
Please, use the API package `@eclipse-che/theia-remote-api` instead of the Che-Server-specific implementation.

Reply:
done